src/assets/scripts/base/PositionModel.js

```diff
@@ -238,15 +204,28 @@ export default class PositionModel {
     }
 
     /**
-     * Fascade for `adjustForMagneticNorth`
-     *
-     * Adjusts x & y coordinates from true north to magnetic north
-     *
+     * Checks whether or not this `PositionModel` has a reference PositionModel
+     * Without the reference position, the rotation due to magnetic variation will not be applied
+     * @for PositionModel
+     * @method _hasReferencePosition
+     * @return {Boolean} whether this position is based on a reference position
+     */
+    _hasReferencePosition() {
+        return this.reference_position !== null;
+    }
+
+    /**
+     * Determine the `x` and `y` values of the `PositionModel`, used for drawing on the canvas
      * @for PositionModel
-     * @method adjustXYForMagneticNorth
+     * @method _calculateScreenPosition
+     * @private
      */
-    adjustXYForMagneticNorth() {
-        const { x, y } = adjustForMagneticNorth(this.x, this.y, this.magnetic_north);
+    _calculateScreenPosition() {
+        if (!this._hasReferencePosition()) {
```

Comment:
this same logic could be done with `!this.reference_position`, but I like this abstraction be cause it reads better. 👍